vnet/lib/vnet/openflow/ovs_ofctl.rb

```diff
@@ -37,20 +37,36 @@ def add_flow(flow)
       debug log_format("'#{command}' => #{result}") if verbose
     end
 
+    # TODO: Deprecate.
```

Comment:
Why not deprecate right away by putting a big ugly message in the log? Something like this:

```ruby
logger.warn "######################################################\n"\
            "add_ovs_flow is deprecated! Use add_ovs_flow2 instead!\n"\
            "######################################################\n\n"\
            "add_ovs_flow was used here: #{caller.first}"                   
```